scss/index.scss

```diff
@@ -0,0 +1,126 @@
+@import "imports/variables";
+
+@font-face {
+  font-family: Ketchum;
+  src: url('../fonts/Ketchum.otf');
+}
+
+body {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: 'Cantarell', sans-serif;
+}
+
+#scroll {
+  width: 1000px;
+  background-color: $mine-shaft-grey;
+  height: 700px;
+  margin: 0 auto;
+  overflow: auto;
+}
+
+ul {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+h2 {
+  margin-bottom: 0;
+  margin-top: 2%;
+}
+
+.poke-container {
+  width: 100%;
+  height: 150px;
+  outline-style: solid;
+  outline-offset: -2px;
+}
+
+.info {
```

Comment:
.info and .picture-holder need to add to 100% currently add to 95%

Reply:
done in 
e80e691